src/sentry/snuba/metrics/utils.py

```diff
@@ -155,6 +155,8 @@ def generate_operation_regex():
 
 OP_REGEX = generate_operation_regex()
 
+# Regex that is used to validate that an operation is a percentile operation in the form p**.
+PERCENTILE_OP_REGEX = r"p\d\d"
```

Comment:
```suggestion
PERCENTILE_OP_REGEX = r"^p[0-9]{2}"
```
it's important to add `^` because you are looking for starts from `p`.

```
[0-9]{2}
```
IMO just more readable (but it's very opinionated.)
cc @ahmedetefy

Reply:
I think we don't need a regex, we have a list of predefined operation percentiles that the datasketch supports listed, we can check just on those

Reply:
good catch ^ agreed